README.md

```diff
@@ -0,0 +1,117 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>. The official documentation is https://spectrum-os.org/doc/.
+
+> :balloon: Currently, the cross-compiled version of Spectrum is for i.MX 8QXP board only.
```

Comment:
This branch towards wayland from guest support does NOT support cross-compilation. Also, the wayland from guest support will be only i.MX8 QM board with SMMU HW.